src/fontra/views/fontinfo/panel-sources.js

```diff
@@ -428,13 +452,57 @@ class SourceBox extends HTMLElement {
     }
   }
 
-  toggleShowHide() {
-    const element = this.querySelector("#open-close-icon");
-    element.classList.toggle("item-closed");
+  toggleShowHide(altKey) {
+    const cardElements = !altKey
+      ? [this]
+      : document.querySelectorAll(".fontra-ui-font-info-sources-panel-source-box");
+
+    const thisIconElement = this.querySelector("#open-close-icon");
+    const isClosed = thisIconElement.classList.contains("item-closed");
+
+    for (const cardElement of cardElements) {
+      const elementIcon = cardElement.querySelector("#open-close-icon");
+      if (isClosed) {
+        cardsInfos[cardElement.sourceIdentifier]["isClosed"] = false;
+        cardElement.classList.remove("item-closed");
+        elementIcon.classList.remove("item-closed");
+        for (const child of cardElement.children) {
+          child.classList.remove("min-height");
+        }
+      } else {
+        cardsInfos[cardElement.sourceIdentifier]["isClosed"] = true;
+        cardElement.classList.add("item-closed");
+        elementIcon.classList.add("item-closed");
+        for (const child of cardElement.children) {
+          child.classList.add("min-height");
+        }
+      }
+    }
+  }
+
+  getOneliner() {
+    const onelinerElement = html.div({
+      class: "fontra-ui-font-info-sources-panel-oneliner",
+    });
+    onelinerElement.appendChild(html.span({ class: "bold" }, [this.source.name] + " "));
+
+    for (const axis of this.fontAxesSourceSpace) {
+      const axisElement = document.createElement("span");
+      const sourceLocationValue = round(this.source.location[axis.name], 2);
+      axisElement.innerText = `${axis.name}=${sourceLocationValue}, `;
 
-    for (const child of this.children) {
-      child.classList.toggle("min-height");
+      if (axis.defaultValue == sourceLocationValue) {
+        axisElement.classList.add("default");
+      }
+      onelinerElement.appendChild(axisElement);
+    }
+
+    const lastChild = onelinerElement.lastChild;
```

Comment:
I find this section a bit awkward, and would prefer to not add it in the first place.

One way to do that would be:
```js
    for (const [i, axis] of enumerate(this.fontAxesSourceSpace)) {
      if (i > 0) {
        onelinerElement.append(", ");
      }
      // rest of code that adds axisElement
    }
```

Reply:
ok, I changed it.